docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+A global *mark* GC cycle increment is triggered by an [allocation taxation threshold](vgclog_examples.md#balanced-examples).
+
+The following elements log the GC increments, subincrements and operations of the global *mark* GC cycle:
+
+|GC increment         | GC operations| *stop-the-world* or concurrent| XML element of GC increment| Details                         |
+|---------------------|-------------|-------------------------------|--------------------------------------|-----------------------|
+|`global mark phase` subincrement| mark | *stop-the-world* | `<gc-start>`, `<gc-end>` |The global mark phase operations start at the beginning of the cycle and run through all *regions* until the final *region* |
+|`GMP work packet processing` subincrement| work packet processing (WPP) operations | concurrent and sometimes final operations during a *STW* to complete the subincrement | `<concurrent-start>`, `<concurrent-end>`| The `GMP work packet processing subincrement` runs immediately after the `global mark phase` subincrement |
+|final global mark phase increment | final global mark phase operations including class unload | *stop-the-world* | `gc-start>`, `<gc-end>`| Final increment. Runs the final global mark phase operations followed by operations to finish the cycle  |
+
+The `balanced` global *mark* GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and some child elements are omitted for clarity:
+
+```xml
+<exclusive-start/>                        (STW pause starts)
+
+<allocation-taxation/>                    (memory threshold trigger recorded)
+
+<cycle-start type="global mark phase"/>   (global mark cycle starts)
+
+<gc-start type="global mark phase"/>      (1st GMP STW subincrement starts)
+
+    <mem-info>                            (memory status before operations)
+
+    <remembered-set>
+
+    </mem-info>         
+
+</gc-start> 
+
+<gc-op type="mark increment" />           (STW copy forward operation completed)
+
+<gc-op  type="class unload" />            (optional STW class unload operation completed)
+
+<gc-end>                                  (partial cycle increment ends)
+
+    <mem-info>                            (memory status after operations)
+          
+    <remembered-set>
+
+    </mem-info>
+
+<gc-end>
+
+<concurrent-start type="GMP work packet processing"/> (1st GMP concurrent subincrement starts)
+
+<exclusive-end/>                                      (STW pause ends)
+
+<concurrent-end type="GMP work packet processing"/>   (1st GMP concurrent subincrement ends)
+
+<gc-op type="mark increment"/>                        (copy forward operation runs concurrently)
+
+</concurrent-end type="GMP work packet processing"/>>
+
+...                                       (partial GC cycles run during STW pauses)
```

Comment:
before and after this line you could add extra '...' implying that PGC is just executed for just a portion of time between concurrent-end and next GMP STW subincrement.

optionally you could re-emphaisse that application threads also run
```
</concurrent-end type="GMP work packet processing"/>>
... (optionally you may state: application thread runs)
... (partial GC cycles run during STW pauses)
... (optionally you may state: application thread runs)
<exclusive-start/> (STW pause starts)
```

Reply:
...                                       (application threads run. STW pauses start and  
                                          end to run Partial GC cycles.)